resource/out

```diff
@@ -81,6 +83,15 @@ if [ -n "$version_file" ]; then
   version_arg=--version=$(cat "$version_file")
 fi
 
+has_image_arg=""
+if [ "${has_image}" == "true" ]; then
+  has_image_arg=--has-image
+fi
+if [ "${has_image}" == "false" ]; then
+  has_image_arg=--no-has-image
```

Comment:
Not sure I follow, shouldn't this just be an empty string?  There's no `--no-has-image` param in the release.new function

Reply:
Yeah, the way [invoke](https://www.pyinvoke.org/) (the library we use to generate the shell commands) works is that it's based on some "conventions".
- Things defined in python with an "_" are expected in the shell command with a "-". For example `has_image` -> `has-image`.
- Booleans are special. `--has-image` means `has_image = True`. `--has-image=True` is parsed to a string. Because of that, they also follow the `--no` ""convention"" (👀). So if you want a boolean to be `false` you would use `--no-your-option`. In our case `--no-has-image`.

It also took me a while to understand how to get a False as the documentation is not great. Shout out to chat gpt 🤖

Reply:
Well that is not intuitive at all

Reply:
> Well that is not intuitive at all

As their documentation says, _Welcome to Invoke!_